src/components/CompareResults/ResultsMain.tsx

```diff
@@ -39,11 +39,13 @@ function ResultsMain() {
   const styles = {
     alert: style({
       width: '100%',
+      fontSize: '16px',
     }),
     container: style({
       backgroundColor: themeColor100,
       margin: '0 auto',
       marginBottom: '80px',
+      maxWidth: 'xl',
```

Comment:
Oh, I meant on the `Container` component itself as a prop. However, `xl` is way too large. Could we update the Containers in `ResultsMain` and `SubtestResultsMain` like this:

 <Container maxWidth={false} sx={{maxWidth: '1300px'}} className={styles.container} data-testid='results-main'>